### DISCUSSION TITLE
sending out email everyday

### DISCUSSION THREAD

**Discussion by gl2007** [roleoroleo/yi-hack-Allwinner-v2#881] [Q&A]
Since my spectrum probably will change the router IP address, I want to send myself an email daily using a cron job, which needs to tell me my router external ip address. Is sending email supported in Yi or via this hack? If not but is possible, what will I have to do to get this done or how should I find it if already there?

**Comment by roleoroleo** [comment]
busybox contains sendmail but it's not linked.
Try this:
```
cp /tmp/sd/yi-hack/bin/busybox /tmp/sd/yi-hack/bin/sendmail
/tmp/sd/yi-hack/bin/sendmail
```

**Reply by gl2007** [reply]
Great thanks. I will try to put in a cron job to get external IP and send out email via cron job. Do you other ideas to keep me informed of public IP of router changes if any? Also, in the worst case if the cron job fails or script fails, this should affect the general working of the back right?
